Makefile

```diff
@@ -13,3 +13,18 @@ dev: ## Run local server to check you content while writing
 
 build: ## Build the static files of the website
 	hugo
+
+docker-base: 
+	docker build -f Dockerfile.base -t baker-docs:base .
+
+docker-dev: docker-base ## Use docker for baker website development
+	docker run -w /baker -v $$PWD:/baker -p 1313:1313 baker-docs
+
+docker-build-prod: docker-base
+	docker build -t baker-docs:prod .
+
+docker-run-prod: docker-build-prod ## Build baker website production version and runs hugo server on port 80
+	docker run -p 80:1313 -t baker-docs:prod
```

Comment:
can we use a port different from 80? it fails on my machine as it is used

Reply:
moreover, I'm not sure I'm getting the use of the prod version. It is only on the port 80 instead of 1313 with minified files. Maybe we should add a `build` target to build the website with docker?

Reply:
> can we use a port different from 80? it fails on my machine as it is used

I thought we wanted a production version of the website, bound to port 80, to be deployed on aws inside a docker image.
If that's not what we want, then we can just remove `docker-build-prod` and `docker-run-prod` makefile targets altogether.

> Maybe we should add a build target to build the website with docker?

If we only want to use docker to build the website into a target directory, it's probably enough to bind mount the current directory (as does `docker-dev`).

Reply:
yup. The production version is created with `make build` that creates static files to be uploaded to GH pages

Reply:
Ok so in this case I'm going to remove code and files related to `docker-prod` targets and add `docker-build` instead.
It will just use docker to build the website locally